lib/api/CInferenceModelDefinition.cc

```diff
@@ -278,11 +278,7 @@ std::string CInferenceModelDefinition::jsonString() {
 }
 
 void CInferenceModelDefinition::addToDocument(rapidjson::Value& parentObject,
-                                              TRapidJsonWriter& writer) const { //input
-    rapidjson::Value inputObject = writer.makeObject();
-    m_Input.addToDocument(inputObject, writer);
-    writer.addMember(JSON_INPUT_TAG, inputObject, parentObject);
```

Comment:
`JSON_INPUT_TAG` is no longer needed I'd guess.

Reply:
fixed